conductor/src/main/java/com/bluelinelabs/conductor/Conductor.kt

```diff
@@ -0,0 +1,35 @@
+@file:JvmName("Conductor")
+
+package com.bluelinelabs.conductor
+
+import android.app.Activity
+import android.os.Bundle
+import android.view.ViewGroup
+import androidx.annotation.UiThread
+import com.bluelinelabs.conductor.internal.LifecycleHandler
+import com.bluelinelabs.conductor.internal.ensureMainThread
+
+/**
+ * Point of initial interaction with Conductor. Used to attach a [Router] to your Activity.
+ */
```

Comment:
This comment is not attached to any syntactic entity...

Reply:
Is it possible to add a kind of file description in kdoc?

Reply:
I don't think so. I deleted it for now.